.github/workflows/pr.yaml

```diff
@@ -93,6 +114,30 @@ jobs:
           platforms: ${{ matrix.platforms }}
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+          
+      - name: Run Trivy in table mode for (${{ matrix.name }})
+        env:
+          TRIVY_NON_SSL: true
+          TRIVY_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-db
+        uses: aquasecurity/trivy-action@0.18.0
+        with:
+          image-ref: registry:5000/fb-output-plugin-${{ matrix.name }}
+          format: table
+          ignore-unfixed: true
+          severity: 'MEDIUM,LOW'
+      
+      - name: Fail build on High/Criticial Vulnerabilities
+        env:
+          TRIVY_NON_SSL: true
+          TRIVY_DB_REPOSITORY: public.ecr.aws/aquasecurity/trivy-db
+        uses: aquasecurity/trivy-action@0.18.0
```

Comment:
is this the latest: 0.18.0?

Reply:
No. 0.28.0 is the latest version. I was referring to some repo that has this version. Will update it